MavenAnalysisConf/src/main/resources/rulesets/java/pmd-level0.xml

```diff
@@ -84,7 +75,7 @@
         <description>Non-test classes may not have more than 4 duplicate literals, unless the literal is for an annotation.</description>
         <properties>
             <property name="skipAnnotations" value="true" />
-            <property name="violationSuppressXPath" value="./ancestor::TypeDeclaration//MethodDeclaration[../Annotation/MarkerAnnotation/Name[@Image='Test']]" />
+            <property name="violationSuppressXPath" value="./ancestor-or-self::ClassDeclaration[contains(@SimpleName, 'Test')]"/>
```

Comment:
this effectively does the same thing, only uses the classname to determine whether the rule should be skipped instead of the `@Test` annotation... this means that duplicate literals inside of a class (like `private final fooString = "foostr";` is declared multiple times) would be allowed now, but that isn't a scenario I've ever seen - what you usually see (and this allows) is the same string declared multiple times across individual tests in the same class.